core/pkg/resourcehandler/k8sresources.go

```diff
@@ -313,29 +360,6 @@ func (k8sHandler *K8sResourceHandler) collectRbacResources(allResources map[stri
 	return nil
 }
 
-func getCloudProviderDescription(allResources map[string]workloadinterface.IMetadata, ksResourceMap *cautils.KSResources) (string, error) {
```

Comment:
[Warning]: It looks like you re-named this function to `collectCloudResources`. Notice, by renaming the function you might be overriding the logic we had added to this function.

Reply:
I made some refactoring - collectCloudResources has the outer logic and it calls the different getters of each resource